plentyofpets/lib/components/pet_description.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+// Class used to display and format pet's info on the pet profile page
+class PetDescription extends StatelessWidget {
+  Map<String, dynamic> data = {};
+
+  PetDescription(this.data, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+      Text('${data['name']}', style: Theme.of(context).textTheme.headline5),
+      Text('Type: ${data['type']}'),
+      Text('Breed: ${data['breed']}'),
+      Text('Availability: ${data['availability']}'),
+      Text('Date Created: ${dateConverter(data['timestamp'])}'),
+      const Text('Disposition:'),
+      dispositionCol(data['disposition']),
+    ]);
+  }
+}
+
+// This data converter function takes a timestamp and coverts it to a
+// MM/DD/YY string
```

Comment:
This function looks great, but this feels like it should be in a separate utility class so it's easily reusable

Reply:
Great idea! I've pulled the function out into it's own file in the components folder, then referenced the file in the pet_description. I tested it, and it looks like it still works!